src/plugins/orangehrmBuzzPlugin/test/fixtures/testcases/BuzzCommentAPITestCases.yaml

```diff
@@ -418,3 +418,473 @@ GetOne:
+    exception:
+      class: 'OrangeHRM\Core\Api\V2\Exception\ForbiddenException'
+      message: 'Unauthorized'
+
+  'Delete Comment By Share Id and Comment Id - Not added Comment':
+    userId: 2
+    services:
+      buzz.buzz_service: \OrangeHRM\Buzz\Service\BuzzService
+      core.authorization.user_role_manager_helper: \OrangeHRM\Core\Authorization\Helper\UserRoleManagerHelper
+      pim.employee_service: \OrangeHRM\Pim\Service\EmployeeService
+      admin.user_service: \OrangeHRM\Admin\Service\UserService
+      core.config_service: \OrangeHRM\Core\Service\ConfigService
+    attributes:
+      shareId: 2
+      commentId: 10
+    factories:
+      core.authorization.user_role_manager: [ '\OrangeHRM\Core\Authorization\Manager\UserRoleManagerFactory', 'getNewUserRoleManager' ]
+    exception:
+      class: '\OrangeHRM\Core\Api\V2\Exception\RecordNotFoundException'
+      message: 'Record Not Found'
+
+  'Delete Comment By Share Id and Comment Id - Not added Share':
+    userId: 2
+    services:
+      buzz.buzz_service: \OrangeHRM\Buzz\Service\BuzzService
+      core.authorization.user_role_manager_helper: \OrangeHRM\Core\Authorization\Helper\UserRoleManagerHelper
+      pim.employee_service: \OrangeHRM\Pim\Service\EmployeeService
+      admin.user_service: \OrangeHRM\Admin\Service\UserService
+      core.config_service: \OrangeHRM\Core\Service\ConfigService
+    attributes:
+      shareId: 20
+      commentId: 10
+    factories:
+      core.authorization.user_role_manager: [ '\OrangeHRM\Core\Authorization\Manager\UserRoleManagerFactory', 'getNewUserRoleManager' ]
+    exception:
+      class: '\OrangeHRM\Core\Api\V2\Exception\RecordNotFoundException'
+      message: 'Record Not Found'
+
+Update:
+  'Update Comment By Share Id and Comment Id - ESS User - Own Comment':
+    userId: 3
+    now:
+      datetime: '2022-12-14 05:40:00'
+    services:
+      buzz.buzz_service: \OrangeHRM\Buzz\Service\BuzzService
+      core.authorization.user_role_manager_helper: \OrangeHRM\Core\Authorization\Helper\UserRoleManagerHelper
+      pim.employee_service: \OrangeHRM\Pim\Service\EmployeeService
+      admin.user_service: \OrangeHRM\Admin\Service\UserService
+      core.config_service: \OrangeHRM\Core\Service\ConfigService
+    attributes:
+      shareId: 2
+      commentId: 6
+    factories:
+      core.authorization.user_role_manager: [ '\OrangeHRM\Core\Authorization\Manager\UserRoleManagerFactory', 'getNewUserRoleManager' ]
+    body:
+      text: 'This is edited comment'
+    data: {
+      'comment': {
+        id: 6,
+        createdDate: '2022-10-27',
+        createdTime: '01:00',
+      },
+      'share': {
+        'id': 2
+      },
+      'employee': {
+        'empNumber': 5,
+        'lastName': 'Abel',
+        'firstName': 'Ashley',
+        'middleName': '',
+        'employeeId': '0005',
+        'terminationId': 1,
+      },
+    }
+    meta: [ ]
+
+  'Update Comment with large text (char count = 65530)':
```

Comment:
is this `65530` count correct?
if the char length equals 65530, this test case should pass

Reply:
Previously checked with ENT version. Will check again

Reply:
Checked, char length here is 65531